### PR TITLE
Us29

### PR DESCRIPTION
**Pull request by BJSherman80**
As a registered user
When I visit my Profile Orders page
And I click on a link for order's show page
My URL route is now something like "/profile/orders/15"
I see all information about the order, including the following information:
- the ID of the order
- the date the order was made
- the date the order was last updated
- the current status of the order
- each item I ordered, including name, description, thumbnail, quantity, price and subtotal
- the total quantity of items in the whole order
- the grand total of all items for that order

### CODE REVIEW

File: app/views/user_orders/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  <li>Date order was updated: <%=@order.updated_at %></li>
+  <li>Order status: <%=@order.status %></li>
+  <li>Date order was created: <%=@order.created_at %></li>
+  <li>Date order was created: <%=@order.created_at %></li>
+  <li>Date order was created: <%=@order.created_at %></li>
+  <li>Items: <% @order.items.each do |item|%></li>
+    <li>Item name: <%= item.name %></li>
+    <li>Description: <%= item.description %></li>
+    <li>Image: <%= item.image %></li>
+    <li>Quantity: <%= item.item_orders.first.quantity %></li>
```
Comment (evilaspaas1) [inline app/views/user_orders/show.html.erb:13]: we'll want to break this out into the model.

File: spec/features/orders/show_spec.rb [hunk 2]
```diff
@@ -35,6 +35,7 @@
       expect(page).to have_content(@pencil.description)
       expect(page).to have_content(@pencil.price)
       expect(page).to have_content(@pencil.item_orders.first.subtotal)
+      expect(page).to have_content(@tire.item_orders.first.subtotal)
```
Comment (nicorithner) [inline spec/features/orders/show_spec.rb:38]: we should refactor this line later on so it is not so long

**Review by nicorithner — APPROVED** [review]

File: app/views/user_orders/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    <li>Price: <%= item.price %></li>
+    <li>Subtotal: <%=item.item_orders.first.subtotal%></li>
```
Comment (evilaspaas1) [inline app/views/user_orders/show.html.erb:15]: same here

**Review by evilaspaas1 — APPROVED** [review]

File: app/views/user_orders/show.html.erb (hunk 1 above)
Comment (nicorithner) [reply app/views/user_orders/show.html.erb:13]: I agree. I think this belongs to the module

**Pull request merged**